.circleci/config.yml

```diff
@@ -440,7 +440,6 @@ jobs:
       - image: circleci/android:api-28-ndk
     steps:
       - checkout
-      - run: ./gradlew --no-daemon lint
```

Comment:
Actually I think this needs to be left in. If I remember correctly it actually does something different than ktlint and detekt